mantle/kola/tests/misc/raid.go

```diff
@@ -173,7 +176,17 @@ func checkIfMountpointIsRaid(c cluster.TestCluster, m platform.Machine, mountpoi
 // is found to be mounted at /.
 func checkIfMountpointIsRaidWalker(c cluster.TestCluster, bs []blockdevice, mountpoint string) bool {
 	for _, b := range bs {
+		singleMnt, listMnt := false, false
 		if b.Mountpoint != nil && *b.Mountpoint == mountpoint {
+			singleMnt = true
+		} else if len(b.Mountpoints) != 0 {
+			for _, mnt := range b.Mountpoints {
+				if mnt != nil && *mnt == mountpoint {
+					listMnt = true
+				}
+			}
+		}
+		if singleMnt || listMnt {
```

Comment:
Minor/optional: would probably be cleaner to just factor out the bits that look for the mountpoint into a separate function where they can just `return true` and then e.g. `if blockdev_has_mountpoint(...) {`